domains/bimarshaw.json

```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "Bimarsha69",
+        "email": "bimarshathapa2@gmail.com"
+    },
+
+    "record": {
+        "A": ["info.bimarshathapa.com.np"]
```

Comment:
Invalid A record. A records must be IPv4.

Reply:
Please use a CNAME instead.